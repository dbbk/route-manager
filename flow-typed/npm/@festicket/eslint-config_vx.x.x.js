// flow-typed signature: 564086c0274f153b4c37ac5c954f219e
// flow-typed version: <<STUB>>/@festicket/eslint-config_v^1.1.5/flow_v0.63.1

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@festicket/eslint-config'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@festicket/eslint-config' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module '@festicket/eslint-config/index' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
declare module '@festicket/eslint-config/index.js' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
