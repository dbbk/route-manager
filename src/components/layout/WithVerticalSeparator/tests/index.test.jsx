// @flow

import React from 'react';
import { mount } from 'enzyme';
import { WithVerticalSeparator } from '@festicket/react-ui-components';

describe('<WithVerticalSeparator />', () => {
  test('with initial props', () => {
    const tree = mount(
      <WithVerticalSeparator>
        <span>this is some text</span>
      </WithVerticalSeparator>,
    );
    expect(tree).toMatchSnapshot();
  });
});
