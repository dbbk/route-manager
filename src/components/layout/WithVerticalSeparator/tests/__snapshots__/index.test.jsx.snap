// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WithVerticalSeparator /> with initial props 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0::after {
  display: inline-block;
  content: '';
  width: 1px;
  height: 20px;
  margin: 0 20px;
}

@media (min-width:568px) {
  .c0::after {
    margin: 0 30px;
  }
}

<Styled(WithVerticalSeparator)>
  <WithVerticalSeparator
    className="c0"
  >
    <span
      className="c0"
      key=".0"
    >
      this is some text
    </span>
  </WithVerticalSeparator>
</Styled(WithVerticalSeparator)>
`;
